config/initializers/mime_types.rb

```diff
@@ -3,3 +3,4 @@
 # Add new mime types for use in respond_to blocks:
 # Mime::Type.register "text/richtext", :rtf
 # Mime::Type.register_alias "text/html", :iphone
+Mime::Type.register "application/x-gzip", :xml_gz, [], ["xml.gz"]
```

Comment:
Is this needed anymore? The Rack::Deflater handles now the gzipping, and in the response we send:

```
Content-Encoding: gzip
Content-Type: application/xml
```

Full cURL:

```
➜  ~ curl 'http://test.lvh.me:3000/sitemap.xml' -H 'Pragma: no-cache' -H 'Accept-Encoding: gzip, deflate, sdch' -H 'Accept-Language: en-US,en;q=0.8,fi;q=0.6,de;q=0.4,fr;q=0.2' -H 'Upgrade-Insecure-Requests: 1' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36' -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8' -H 'Cache-Control: no-cache' -H 'Cookie: _sharetribe_session_newnewnew=2194b4c4245929eb8ac04fb7ae4ebf50; _sharetribe_session_newnewnew4=814a4b5ea25b5777676c9b8c5724415f; _st_session=c9d6795069a8a65b41f936ff5c1db67e' -H 'Connection: keep-alive' -i
HTTP/1.1 200 OK
X-Frame-Options: SAMEORIGIN
X-Xss-Protection: 1; mode=block
X-Content-Type-Options: nosniff
Content-Disposition: attachment; filename="sitemap.xml"
Content-Transfer-Encoding: binary
Content-Type: application/xml
Cache-Control: private
Etag: W/"e0002c666520520bb6e5ded9b894e758"
X-Meta-Request-Version: 0.4.0
X-Request-Id: 35c92e0a-d63f-4b7f-82f6-541dcb6116ce
X-Runtime: 0.048266
Vary: Accept-Encoding
Content-Encoding: gzip
Server: WEBrick/1.3.1 (Ruby/2.3.1/2016-04-26)
Date: Thu, 01 Sep 2016 13:57:19 GMT
Content-Length: 467
Connection: Keep-Alive
Set-Cookie: _st_session=c9d6795069a8a65b41f936ff5c1db67e; path=/; expires=Sat, 01 Oct 2016 13:57:19 -0000; HttpOnly
```

So to my understanding we can remove this, right?